www-plugins/passff/passff-1.1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_4 )
+
+inherit python-single-r1
+
+DESCRIPTION="zx2c4 pass manager extension for Firefox (Host Binary)"
+HOMEPAGE="https://github.com/passff/passff"
+SRC_URI="
+	https://github.com/passff/passff/releases/download/${PV}/passff.json -> ${P}.json
+	https://github.com/passff/passff/releases/download/${PV}/passff.py -> ${P}.py
+"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+TARGET_DIR="usr/lib/mozilla/native-messaging-hosts"
+
+S="${WORKDIR}"
+
+src_unpack() {
+	cp "${DISTDIR}/${P}.json" . || die
+	cp "${DISTDIR}/${P}.py" . || die
+}
+
+src_compile() {
+	sed -i "s|PLACEHOLDER|${EPREFIX}/${TARGET_DIR}/${PN}.py|" "${P}.json" || die
+}
+
+src_install() {
+	insinto "${EPREFIX}/${TARGET_DIR}"
```

Comment:
Don't include EPREFIX here, built-in helpers prepend it for you. So just `/${TARGET_DIR}`. In other words, it would be probably cleaner if you included the slash inside TARGET_DIR, and removed it from here and sed.